<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bind Voice &amp; Cmds</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,3">
     <item>
      <widget class="QLabel" name="add">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>57</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>UpArrowCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Adobe Devanagari&quot;;
color: rgb(0, 170, 255);
</string>
       </property>
       <property name="text">
        <string>Bind voice commands to program exe file paths</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,5,1">
         <item alignment="Qt::AlignLeft">
          <widget class="QLabel" name="add_3">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string notr="true">app.exe path:      </string>
           </property>
           <property name="textFormat">
            <enum>Qt::PlainText</enum>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <property name="indent">
            <number>-9</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="pathEdit"/>
         </item>
         <item>
          <widget class="QToolButton" name="pathButton">
           <property name="text">
            <string notr="true">...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>78</width>
             <height>48</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="addButton">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 170, 255)</string>
           </property>
           <property name="text">
            <string>Add</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="returnButton">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(170, 255, 255)</string>
           </property>
           <property name="text">
            <string>Return</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,6">
         <item>
          <widget class="QLabel" name="add_2">
           <property name="text">
            <string>voice cmd(English):</string>
           </property>
           <property name="indent">
            <number>-1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="cmdText"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
